source/codegen/metadata/nidcpower/functions.py

```diff
@@ -355,6 +355,79 @@
     ],
     'returns': 'ViStatus'
   },
+  'CreateAdvancedSequenceWithChannels': {
```

Comment:
@jonlugoatx , currently I manually tested these 2 APIs using the python client code for a simulated DCPower device. But should I also add some system tests for these APIs like:
- Call CreateAdvancedSequence with sequence already existing -> Gives error
- Call CreateAdvancedSequence -> delete that sequence -> Call CreateAdvancedSequence again -> Works fine

Reply:
I'll defer to @ganeshnie about the level of testing. I'm not familiar enough with the DC-Power API to know what would give us the best testing value. We should have at least one test that successfully transfers an `Int32[]` without crashing.

Reply:
@jonlugoatx , I discussed with @ganeshnie and we decided to have a system test for ImportAttributeConfigurationBuffer and ExportAttributeConfigurationBuffer APIs which would cover both the changes made to mako file.